analyzers/its/sources/ManuallyAddedNoncompliantIssues.CS/IntentionalFindings/S6675.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace IntentionalFindings
+{
+    public class S6675
+    {
+        public void Method(TraceSwitch traceSwitch)
+        {
+            Trace.WriteLineIf(traceSwitch.TraceError, "Message"); // Noncompliant (S6675) {{Don't use Trace.WriteLineIf with TraceSwitch level checks.}}
```

Comment:
This one needs to be modified as well.

Reply:
I removed the message (it's ignored anyway).